service/src/test/java/skills/intTests/WebsocketSpecs.groovy

```diff
@@ -116,6 +119,37 @@ class WebsocketSpecs extends DefaultIntSpec {
         wsResults.find{it.skillId=='badge1'}.completed[0].name == badge.name
     }
 
+    def "Non-notified global badge achievements are notified when user connects to websocket" () {
```

Comment:
good test! can you please add a test with multiple users and multiple badges to verify that right messages arrive to the right users?